examples/tester.rs

```diff
@@ -0,0 +1,309 @@
+//! Basic winit interactivity example.
```

Comment:
Nit: Maybe call the example something like `interactivity.rs` or `drawing_app.rs`?

Reply:
Also, you can group stuff in a folder, like `examples/drawing_app/main.rs` + `examples/drawing_app/index.html`